packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -416,15 +416,15 @@ export class EngineReportingAgent<TContext = any> {
       },
     ).catch((err: Error) => {
       throw new Error(
-        `Error sending report to Apollo Engine servers: ${err.message}`,
+        `Error sending report to Apollo Graph Manager servers: ${err.message}`,
       );
     });
 
     if (response.status < 200 || response.status >= 300) {
       // Note that we don't expect to see a 3xx here because request follows
       // redirects.
       throw new Error(
-        `Error sending report to Apollo Engine servers: HTTP status ${
+        `Error sending report to Apollo Graph Manager servers: HTTP status ${
```

Comment:
Same concern as my previous comment.

Reply:
This is a message that explicitly comes with dynamic text in it, for example. If upstream changes, surely we wouldn't bump to a major version. From the users' perspective, this entire message might as well come from our BE.

Reply:
@zionts Same reply from https://github.com/apollographql/apollo-server/pull/3276#discussion_r332470442.  :smile: